libs/stocks/feature-shell/src/lib/stocks/stocks.validator.ts

```diff
@@ -0,0 +1,28 @@
+import { Validators, FormControl, FormGroup } from '@angular/forms';
+
+export class CustomDateValidator extends Validators {
```

Comment:
Custom date validator to show error messages when to date is not greater than from date . Selecting an invalid range  makes both dates equal.